bin/capsule.js

```diff
@@ -869,11 +888,59 @@ const addFilesToS3Bucket = async (projectName, bucketName) => {
   });
 }
 
+/**
+ *
+ * @method getCloudFrontDistID
+ *
+ * @param {String} bucketName
+ *
+ * @return {String} distId
+ *
+ */
+const getCloudFrontDistID = async (bucketName) => {
+  let params = {};
+  return new Promise((resolve, reject) => {
+    cfr.listDistributions(params, function(err, data) {
+      if (err) {
+        logIfVerbose(`${err} , ${err.stack}`);
+        reject(undefined)
+      } else {
+        resolve(extractDistId(data, bucketName));
+      }
+    });
+  })
+}
+
+/**
+ * Search through object and get id
+ * of CloudFront distribtuion associated
+ * with the S3 bucket.
+ *
+ * @method extractDistId
+ *
+ * @param {Object} data
+ * @param {String} bucketName
+ *
+ * @return {String} distId
+ *
+ */
+const extractDistId = async (data, bucketName) => {
```

Comment:
Note that you don't need to specify an `async` function if it does not use `await` inside it.

Reply:
Fix being added to stylistic_fixes branch